checkbox-provider-kivu/bin/gpu-load.py

```diff
@@ -29,22 +29,26 @@ def compute_avg_intel(fname : str) -> float:
         else:
             data = json.loads(content)
```

Comment:
The semantic of the module and returned value now change a little bit depending on the argument, i would prefer an other script for that, but if we still want to keep only one file, there will be a need to document it a little bit